LazarevDV/lab_1/calculator.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+
+int main()
+{
+    setlocale(LC_ALL, "Russian");
+    long double secondvalue, firstvalue;
+    char sign, check;
+    cout << "Добро пожаловать в калькулятор Дениса Лазарева!\n";   
+    do { 
+    cout << "Введите два числа и знак операции:\n";
+    cin >> firstvalue >> secondvalue >> sign;
+    switch (sign) {
+    case '+': cout << firstvalue << " + " << secondvalue << " = " << firstvalue + secondvalue << endl; break;
```

Comment:
Please extract `cout << firstvalue << " ? " << secondvalue << " = " << firstvalue ? secondvalue << endl;` from switch..case in order to remove code duplication